src/commands/upgrade-2.0

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+readonly SCRIPT_PATH="/usr/local/bin/upgrade-2.0.0.sh"
+
+function print_help() {
+  echo "2.0 k8s migration script."
+  echo "This script outputs migrated values file on stdout (and saves it in new_values.yaml in current working directory)."
+  echo "Log produced by the migration script are available on stderr."
+  echo
+  echo "Usage:"
+  echo "  upgrade-2.0 <VALUES_FILE.yaml>"
+}
+
+function err() {
+  echo "${1}" >&2
+}
+
+if [[ ${#} -ge 1 && "${1}" == "--help" ]]; then
+  print_help
+  exit 0
+fi
+
+FILE=""
+if [[ ${#} -eq 0 ]]; then
+  readonly STDIN_WAIT_S="${STDIN_WAIT_S:-10}"
+  # Take data from stdin if available and put into temporary file
+  readonly TMPFILE="$(mktemp /tmp/values.yaml.XXXXXX)"
+  # Kubectl can take some time before stdin is available for reading
+  # thats why we check if just before it's required
+  if read -t "${STDIN_WAIT_S}" REPLY; then
+    # Save first line read from stdin
+    echo "${REPLY}" > "${TMPFILE}"
+    # Save rest of the stdin
+    cat <&0 >> "${TMPFILE}"
+  fi
+
+  if [[ ! -s "${TMPFILE}" ]]; then
+    err "Values file was not provided on stdin (or it was provided empty). Aborting."
+    exit 1
+  fi
+
+  FILE="${TMPFILE}"
+else
+  FILE="${1}"
+  if [[ ! -f "${FILE}" ]]; then
+    err "Provided file \"${FILE}\" doesn't exist"
+    exit 1
+  fi
+fi
+
+readonly LOG_FILE="$(mktemp /tmp/upgrade.log.XXXXXX)"
+readonly EXIT_STATUS=$("${SCRIPT_PATH}" "${FILE}" >"${LOG_FILE}" 2>&1; echo $?)
```

Comment:
Could we just redirect stdout to stderr? Technically it would be better experience if script flushes output line by line and not everything at the end

Reply:
Done. I thought that catching and then printing the output would be better but meh.